Library/Homebrew/extend/pathname.rb

```diff
@@ -331,7 +331,6 @@ def verify_checksum(expected)
     raise ChecksumMismatchError.new(self, expected, actual) unless expected == actual
   end
 
-  # FIXME: eliminate the places where we rely on this method
```

Comment:
It's used hilariously often in literally the majority of Homebrew's files and a lot of formulae. At this point it's a part of the public API. (I did try and remove it and quickly gave up when I realised the magnitude).